packages/client/hmi-client/src/App.vue

```diff
@@ -77,7 +77,12 @@ watch(
 		@show-data-explorer="dataExplorerActivated = true"
 	/>
 	<main>
-		<Sidebar v-if="isSidebarVisible" class="sidebar" data-test-id="sidebar" />
+		<Sidebar
+			v-if="isSidebarVisible && project"
```

Comment:
I don't understand the need to add this?
Why not adding it in `isSidebarVisible` test that is built for it.

Reply:
This is a linter error because project can be null, it isn't really the same check semantically as whether the route is valid or not which is what `isSidebarVisible` is checking.

It is more hassle to check at the children components if the project-property is null, and makes a lot more sense for them to be able to make non-null assumptions.